### ISSUE TITLE
TODO: refresh the symbiotic2cs.py

### ISSUE THREAD

**Issue by jamartis** [aufover/rpm-symbiotic#1]
There have been some updates to symbiotic's output messages. Check the $(SUBJ)  script and update it accordingly.

**Comment by lzaoral** [comment]
* Symbiotic's result is `true` when there are no issues. We should fail if there is no `true` or any other matches in the output, so that we know that this output is not supported by `symbiotic2cs` at the moment.
* Symbiotic's possible results are `error\s*\(reason\)`, `ERROR\s*\(reason\)`, `true`, `false\s*\(reason\)` and `unknown\s*\(reason\)`.
* We should integrate staticafi/symbiotic#195 when it's merged.
* `symbiotic2cs` does not generate correct output whenever Symbiotic returns `ERROR` as a result:
```console
$ touch empty.bc; symbiotic empty.bc 2>&1 | symbiotic2cs | csgrep
-:2: parse error: wrong token: T_UNKNOWN (expected T_EVENT)
-:2: parse error: wrong token: T_NULL (expected T_EVENT)
-:2: parse error: failed to guess key event
```
* Test suite for the converter would be nice (both Divine and CBMC have one).

**Comment by lzaoral** [comment]
staticafi/symbiotic#195 was merged so to see the added `Argv:` line, run `symbiotic` with `--debug=all` argument.

**Comment by lzaoral** [comment]
* ~~Print all paths as absolute ones. See https://github.com/aufover/rpm-divine/blob/5d6d2811d0d5beb0425e91f16b408d3e60c99606/divine2csgrep.py#L75 for inspiration.~~

Fixed in bf4404c.
